lib/locales/es.ts

```diff
@@ -64,7 +64,7 @@ export const locale: Locale = {
     black: 'negro',
     gold: 'dorado',
     green: 'verde',
-    purple: 'modaro',
+    purple: 'morado',
```

Comment:
It might be a good idea to add a source. I suggest to use GIRM where in 346d, there is defined the liturgical usage of the colour.

I suggest to use `// src: GIRM, 346d (https://www.santaclaradeestella.es/BIBLIOTECA/ENCICLICAS/INSTRUCCI%C3%93N_GENERAL_DEL_MISAL_ROMANO.htm)`, i.e. in the following format: `// src: GIRM, $paragraf[$point] ($url_in_a_locale)`.

You could do the same for the rest of the colours.